src/frontend/src/components/RequestProgressSidesheet/index.tsx

```diff
@@ -75,6 +75,14 @@ function InvalidRequestProgressItem<TRequest>({
     onRemove,
 }: FailedRequestItemProps<TRequest>) {
     const ignoreTooltipRef = useTooltipRef('Ignore');
+
+    let errors: string[] = [];
```

Comment:
Is this the same as `const errors = error.errors ? Object.values(error.errors) : []`;?

Reply:
Looks like the error.errors[index] is an array? So more like `const errors = Object.values(error.errors || []).reduce((all, e) => ([ ...all, ...e ]), []);` to flatten the array of arrays?